### ISSUE TITLE
Validate .env variables before using inside config services

### ISSUE THREAD

**Issue by CllDMR** [acikkaynak/afetlojistik-backend#177]
.env değerlerinin eksik olması, yanlış formatta olması bu backend uygulamasının runtime'ında hata oluşturmasına neden olabilir. Bu değerler henüz uygulama initialize edilir iken kontrolden geçirilebilir. Bu sayede app ayağa kalmadan env değerlerindeki hataları yakalayabiliriz.
Bu konuyu implement ederken Nestjs [https://docs.nestjs.com/techniques/configuration#schema-validation](dokümantasyonu) yardımcı olacaktır.

**Comment by muraterzioglu** [comment]
Merhabalar, bunun üzerine yardımcı olabilirim. [app.module.ts](https://github.com/acikkaynak/afetlojistik-backend/blob/main/src/app.module.ts) ve [configuration.ts](https://github.com/acikkaynak/afetlojistik-backend/blob/main/src/config/configuration.ts) dosyaları bütün konfigrasyonu içeriyor değil mi?

**Comment by muraterzioglu** [comment]
@CllDMR bu issue ile bearber #178 için de çalışabilir aynı zamanda Dockerfile'a da .env ile ilgili eklemeler yapmak isterim

**Comment by CllDMR** [comment]
Merhabalar, evet #178 ile yakın konular. Dockerfile için ayrı bir issue açıp oradan takip edebiliriz. Issue'ları olabildiğince küçük tutup; kolay açıklanabilir, az efor gerektiren şekilde tutmak daha sağlıklı olabilir.

**Comment by muraterzioglu** [comment]
@CllDMR Çok sevinirim, docker için öneri şeklinde de yeni bir issue açabilirim 👍 Doğrudan pull edip branch atıp pull request mi atıyorum katkı için?

**Comment by CllDMR** [comment]
Evet, şuan için main branch'te iken pull edip, daha sonra main branch'i kullanarak yeni branch oluşturabilirsin. Branch isimlendirmeleri ve commitlint hakkında şuan belirlenmiş bir format yok.

**Comment by muraterzioglu** [comment]
Tamamdır teşekkür ederim!

**Comment by busebatman** [comment]
> Issue'yu sana assign etmek için bende şu an yeterli yetki yok. Yetkisi olan biri bulup isteğini iletebilirim.

@yldrmzffr bilgin var mı?

**Comment by CllDMR** [comment]
Benim önerim branching olarak [trunk-based](https://trunkbaseddevelopment.com/) strategy, commit'lerde de [commitlint](https://commitlint.js.org/#/) formatıdır. Ancak bunlar başka issue'ların konusu :)
